src/provider/store/Actions.ts

```diff
@@ -5,9 +5,19 @@ import {CollectionMap} from '../model/database/Database';
 import {SettingsMap} from '../model/StoredSetting';
 
 import {RootState} from './State';
-import {StoreAPI, AsyncAction, Action} from './Store';
+import {StoreAPI, Action} from './Store';
 
-export class CreateNotification extends AsyncAction<RootState> {
+export interface Actionable {
+    storeAPI: StoreAPI<RootState, RootAction>;
+}
+
+export const enum ToggleCenterVisibilitySource {
```

Comment:
Is this meant to be in here? This will conflict with my PR which adds this

Reply:
Ah stashed that by accident while I was working on your branch.